web/src/locales/fr.json

```
@@ -128,11 +136,12 @@
     "outage-details": "Voir les détails",
     "ESTIMATED_FORECASTS_HIERARCHY": {
       "body": "Les données pour cette zone sont estimées à l'aide d'une combinaison de prévisions de production internes et externes et de moyennes de Time Slicer. Cela vise à fournir une représentation plus précise que la moyenne de Time Slicer seule, mais les données peuvent toujours être sujettes à des inexactitudes. Les données seront mises à jour dès que les données en temps réel seront disponibles.",
-      "pill": "Estimé",
+      "pill": "Estimées",
       "title": "Les données sont estimées"
     },
     "ESTIMATED_TIME_SLICER_AVERAGE": {
       "title": "Les données sont temporaires",
+      "pill": "Estimées",
```

Should this not be:
```suggestion
      "pill": "Préliminaire",
```
It's preliminary in the English version.

(that translation might not be fully correct, feel free to correct it)